.github/workflows/.editorconfig

```diff
@@ -0,0 +1,2 @@
+[*.yml]
```

Comment:
On an unrelated note don't we need on the main editorconfig file to have a different config for makefile that will use tabs?

Maybe we could have this change on this PR..

Reply:
Fixed on master